public/stylesheets/style.css

```diff
@@ -1,312 +1,316 @@
-	font-weight: 500;
-}
-.create-blog-form {
-	background-color: rgb(192, 192, 192, 0.8);
-	margin: 0 auto;
-	width: 40%;
-	border-radius: 10px;
-	font-weight: 500;
+  background-color: rgb(192, 192, 192, 0.8);
+  margin: 0 auto;
+  max-width: 275px;
+  border-radius: 10px;
+  font-weight: 500;
 }
 .field {
-	text-align: center;
-	padding: 4px;
+  text-align: center;
+  padding: 4px;
 }
 input {
-	padding: 0.5%;
-	border-radius: 5px;
-	border: 1px solid #ccc;
+  padding: 0.5%;
+  border-radius: 5px;
+  border: 1px solid #ccc;
 }
 
 input:focus {
-	outline: none;
-	border: 1px solid #3486eb;
+  outline: none;
+  border: 1px solid #3486eb;
 }
 .btn {
-	background-color: darkblue;
-	transition: background-color 0.1s;
-	color: whitesmoke;
-	border-radius: 6px;
-	padding: 2px 6px;
-	font-size: 1rem;
+  background-color: darkblue;
+  transition: background-color 0.1s;
+  color: whitesmoke;
+  border-radius: 6px;
+  padding: 2px 6px;
+  font-size: 1rem;
 }
 
 .btn:hover {
-	background-color: rgb(85, 169, 243);
+  background-color: rgb(85, 169, 243);
 }
 .top-nav {
-	text-align: end;
-	width: 100%;
+  text-align: end;
+  width: 100%;
 }
 .post {
-	background-color: rgb(192, 192, 192, 0.8);
-	color: whitesmoke;
-	border-radius: 15px;
-	margin: 20px;
+  background-color: rgb(192, 192, 192, 0.8);
+  color: whitesmoke;
+  border-radius: 15px;
+  margin: 20px;
 }
 .link-title {
-	font-size: 1.5rem;
-	font-weight: bolder;
-	color: whitesmoke;
-	text-align: center;
+  font-size: 1.5rem;
+  font-weight: bolder;
+  color: whitesmoke;
+  text-align: center;
 }
 .list-item {
-	padding: 1%;
-	margin: 0.5% 5%;
-	background-color: rgba(158, 205, 247, 0.8);
-	border-radius: 10px;
-	font-size: 1rem;
-	color: rgb(29, 37, 37);
-	text-align: center;
-	font-weight: 500;
-	list-style: none;
+  padding: 1%;
+  margin: 0.5% 5%;
+  background-color: rgba(158, 205, 247, 0.8);
+  border-radius: 10px;
+  font-size: 1rem;
+  color: rgb(29, 37, 37);
+  text-align: center;
+  font-weight: 500;
+  list-style: none;
 }
 
 .list-item:nth-child(odd) {
-	background-color: rgb(192, 192, 192, 0.8);
+  background-color: rgb(192, 192, 192, 0.8);
 }
 .post-card {
-	display: flex;
-	flex-direction: column;
-	align-content: center;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
 }
 .post-card a {
-	text-align: center;
+  text-align: center;
 }
 ol {
-	padding: 0;
+  padding: 0;
 }
 .home {
-	max-width: 600px;
-	margin: 0 auto;
+  max-width: 600px;
+  margin: 0 auto;
 }
+
 h4 {
-	background-color: rgb(245, 245, 245, 0.8);
-	margin: 10px;
-	border-radius: 15px;
-	color: darkblue;
-	text-align: center;
-	font-weight: 700;
-	padding: 2rem;
-	font-size: 1.8rem;
-}
-#map {
-	height: 400px; /* The height of div element containing map */
-	width: 100%; /* The width of web page */
+  background-color: rgb(245, 245, 245, 0.8);
+  margin: 10px;
+  border-radius: 15px;
+  color: darkblue;
+  text-align: center;
+  font-weight: 700;
+  padding: 2rem;
+  font-size: 1.8rem;
+}
+
```

Comment:
I think we need to keep the #map here from Christine's merge

Reply:
I think I got trigger happy when I was resolving the conflict, are you able to add these back when you review?